packages/web-forms/e2e/build/style.test.ts

```diff
@@ -4,7 +4,7 @@ test('Build includes component-defined styles', async ({ page }) => {
 	await page.goto('http://localhost:5174/');
 
 	// This ensures that the application is loaded before proceeding forward.
-	await expect(page.getByText('Demo Forms').first()).toBeVisible();
+	await expect(page.getByText('ODK Web Forms Preview').first()).toBeVisible();
```

Comment:
This test executes on the demo page (production mode), which doesn't load the Demo Forms section.
It has been broken for a long time. This fix was initially done [in this PR ](https://github.com/getodk/web-forms/pull/313/files#diff-5284375d1a6d973860f59304d66e1a9a67204b4ac54a9fff2e15133e89b866c5)(to be merged).